truenas/webui
# NAS-125701 / 24.04 / Audit Page QA improvements

Testing of improvements: (see ticket or below)
_p.s I've added a couple of Ukrainian 🇺🇦  translations, so you can check using this beautiful language._

- Add autofocus on the Basic search & Advanced Search when switching between them.
- Submit Basic & Advanced Search by clicking Enter
- Allow switch to basic & switch to advanced to be clicked by Enter
- Fix first column SERVICE (to use translated value) from auditServiceLabels.has()
- Localise Advanced autocomplete placeholder, currently it’s static 'Service = "SMB" AND Event = "CLOSE"'
- Localise Event data column, currently values are not marked for translations
- Fix calendar Week Days color, now it’s DARK on the DARK background
- Hide Export as CSV button if no query results
- Add autocomplete suggestions for “Middleware” service
- Fix Event Data search, it's not working
- Fix TODO --> // TODO: Issue: reset icon will not trigger table update
- Hide calendar in advanced autocomplete input upon SEARCH request (currently it stays active until I move focus in the input)
- Remove Unused `ViewConfigurationState` from `/audit/store/view-configuration.store.ts`
